.env.template

```
@@ -27,6 +27,30 @@ MODELS=`[
       }
     ]
   },
+  {
+    "name" : "HuggingFaceH4/zephyr-orpo-141b-A35b-v0.1",
+    "tokenizer": "HuggingFaceH4/zephyr-orpo-141b-A35b-v0.1",
+    "description": "Zephyr 141B-A35B is a fine-tuned version of Mistral 8x22B, trained using ORPO, a novel alignment algorithm.",
+    "modelUrl": "https://huggingface.co/HuggingFaceH4/zephyr-orpo-141b-A35b-v0.1",
+    "websiteUrl": "https://huggingface.co/HuggingFaceH4/zephyr-orpo-141b-A35b-v0.1",
+    "logoUrl": "https://huggingface.co/datasets/huggingchat/models-logo/resolve/main/zephyr-logo.png",
+    "parameters": {
+      "truncate" : 24576,
+      "max_new_tokens" : 8192,
+    },
+    "promptExamples" : [
+      {
+        "title": "Write an email from bullet list",
+        "prompt": "As a restaurant owner, write a professional email to the supplier to get these products every week: \n\n- Wine (x10)\n- Eggs (x24)\n- Bread (x12)"
+      }, {
+        "title": "Code a snake game",
+        "prompt": "Code a basic snake game in python, give explanations for each step."
+      }, {
+        "title": "Assist in a task",
+        "prompt": "How do I make a delicious lemon cheesecake?"
```

This is a pretty impressive example from @osanseviero 

```suggestion
        "prompt": "Kayley has three brothers. Each of her brothers has two sisters. How many sisters does Kayley have?"
```

---

This is an underdefined question.

Either Kayley is a male (or unidentified) and then she has 2 sisters like all her brothers.
Or Kayley is female and then she is a common sister of all her brothers, which means she has 1 sister.

Internet is wrong.
The dress is blue and black.